dev/TeachingTip/TeachingTip.h

```diff
@@ -293,4 +293,6 @@ class TeachingTip :
 
     //Ideally this would be computed from layout but it is difficult to do.
     static constexpr float s_tailOcclusionAmount = 2;
+    void ClosePopupOnUnloadEvent(winrt::IInspectable const&, winrt::RoutedEventArgs const& e);
+
```

Comment:
Also this method declaration belongs with the other method declarations for event callbacks, somewhere around line 127.

Reply:
Ah yes makes. I will change that later.